tc-config-parser/src/main/java/org/terracotta/config/TCConfigurationParser.java

```diff
@@ -212,10 +212,12 @@ private static void initializeTsaGroupPort(Server server) {
 
   private static void initializeNameAndHost(Server server) {
     if (server.getHost() == null || server.getHost().trim().length() == 0) {
-      if (server.getName() == null) {
-        server.setHost("%i");
-      } else {
+      if (server.getName() != null) {
         server.setHost(server.getName());
+      } else if (!server.getBind().equals(WILDCARD_IP)) {
```

Comment:
Its better to use `WILDCARD_IP.equals(server.getBind())` even if `getBind()` can't be null.